lib/m2ts/metadata-stream.js

```diff
@@ -91,8 +91,6 @@ var
 MetadataStream = function(options) {
   var
     settings = {
-      debug: !!(options && options.debug),
```

Comment:
debug was only used for this console.log, now we just trigger an event at any time and the user can choose what to do.